### PR TITLE
Bump version for official release

### PR DESCRIPTION
**Pull request by avanbrunt-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

CBAPI-2137

## Pull Request Description

Various version and date bumps with alpha message removals

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

Pushing to test pypi

### CODE REVIEW

**Review by kebringer-cb — APPROVED** [review]

**Review by abowersox-cb — APPROVED** [review]
Let's light this candle!

File: docs/conf.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 author = 'Developer Relations'
 
 # The full version, including alpha/beta/rc tags
-release = '1.0a'
+release = '1.0.0'
```
Comment (kebringer-cb) [inline docs/conf.py:26]: file docs/changelog.rst need updating to list the new APIs added

**Review by kebringer-cb — CHANGES_REQUESTED** [review]

File: docs/index.rst [hunk 2]
```diff
@@ -6,9 +6,6 @@
 CBC SDK: Carbon Black Cloud SDK for Python
 ==========================================
 
-In Alpha Development
```
[the remark this answers is not in the document]
Comment (avanbrunt-cb) [reply docs/index.rst:9]: Sounds good. Resolving

File: docs/index.rst [hunk 3]
```diff
@@ -6,9 +6,6 @@
...
---------------------
-
```
[the remark this answers is not in the document]
Comment (avanbrunt-cb) [reply docs/index.rst:11]: Replaced

File: docs/conf.py (hunk 1 above)
Comment (avanbrunt-cb) [reply docs/conf.py:26]: done

**Review by llyon-cb — APPROVED** [review]

**Review by kebringer-cb — APPROVED** [review]

**Review by lhilmes-cb — APPROVED** [review]

**Review by kebringer-cb — APPROVED** [review]

**Pull request merged**